dcrwallet.go

```diff
@@ -137,6 +137,9 @@ func walletMain() error {
 			startPromptPass(w)
 		}
 		startWalletRPCServices(w, rpcs, legacyRPCServer)
+		if w.StakeMiningEnabled {
+			go startTicketPurchase(w)
```

Comment:
OK, started passing in the default options here.

For toggling ticket purchaser, I think we could have rpc similar to `setgenerate`. Will be adding that.

As for passphrase, would it make sense to let the caller make sure the wallet is unlocked before turning on purchasing? If it's not unlocked, the toggle purchase rpc call can return an error.